packages/core/src/actions.ts

```diff
@@ -563,7 +564,8 @@ export function forwardTo<TContext, TEvent extends EventObject>(
 ): SendAction<TContext, TEvent, AnyEventObject> {
   return send<TContext, TEvent>((_, event) => event, {
     ...options,
-    to: target
+    to: target,
+    sendType: 'forward'
```

Comment:
To do this, we add this little piece of `sendType: 'forward'` metadata to the action object created by `forwardTo(...)`